### DISCUSSION TITLE
How can I actually install html/css/js lsp?

### DISCUSSION THREAD

**Discussion by velllu** [helix-editor/helix#7130] [Q&A] [answered]
I installed `vscode-html-languageserver` however it says `Language server exited` whenever i run `:lsp-restart`

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#html

The package should be `vscode-langservers-extracted` and the language server binary name should be `vscode-html-language-server` instead (`vscode-html-languageserver` is a different package).
